Content.Shared/Damage/Systems/StaminaSystem.cs

```diff
@@ -268,15 +301,6 @@ public bool TryTakeStamina(EntityUid uid, float value, StaminaComponent? compone
         if (!Resolve(uid, ref component, false))
             return;
 
-        var ev = new BeforeStaminaDamageEvent(value);
-        RaiseLocalEvent(uid, ref ev);
-        if (ev.Cancelled)
-            return;
-
-        // Have we already reached the point of max stamina damage?
-        if (component.Critical)
-            return;
-
```

Comment:
If someone's crit they shouldn't be taking more stamina damage so this should stay.